src/item_action.cpp

```diff
@@ -370,6 +370,11 @@ void game::item_action_menu( item_location loc )
         num++;
     }
 
+    // custom footer
+    std::string footer = string_format(_("< [<color_yellow>%s</color>] keybindings >"),
+        ctxt.get_desc("HELP_KEYBINDINGS"));
+    kmenu.footer_text = footer.substr(1, footer.length() - 3);
```

Comment:
What's the purpose of substr here? If you don't need part of the string you can simply remove it from the original string.

Reply:
Hi, if you pass a string without "< " and " >" ([<color_yellow>%s</color>] keybindings), then the string will not be translated. Angle brackets are not very suitable for the footer, so I removed them after the translation.
Clarified translation for 39810 segment "<{%s] Assigned Keys >". You need to change { to [ (For EN-RU - https://www.transifex.com/cataclysm-dda-translators/cataclysm-dda/viewstrings/#ru/0f-cataclysm-dda/316259458?q=text%3Akey).

Reply:
> the string will not be translated

It will, after the change is merged and the string is translated on Transifex.

I don't think it is a good idea to use the substring of an existing string's translation because there's no guarantee that the translated string will follow the same format as the original string. For example, the angle brackets might be replaced by characters with different number of utf-8 bytes or multiple characters in a different language, and taking the substring of the translated text will break it in that case.

Reply:
Thank you for the feedback, I'll try to find a more universal way.